server/webhook_http_test.go

```diff
@@ -628,6 +628,9 @@ func TestWebhookHTTP(t *testing.T) {
 			}
 			model.ParseSlackAttachment(rPost, rAttachments)
 			api.On("CreatePost", mock.AnythingOfType("*model.Post")).Return(rPost, nil)
+			api.On("GetUser", mock.AnythingOfType("string")).Return(&model.User{
+				Username: "test-username",
+			}, nil)
```

Comment:
Curious why no tests are failing with the change in this PR. Can we add a test to assert that the username replace is working correctly?